airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -76,6 +79,14 @@ public class SchedulerApp {
   private static final Duration SCHEDULING_DELAY = Duration.ofSeconds(5);
   private static final Duration CLEANING_DELAY = Duration.ofHours(2);
   private static final ThreadFactory THREAD_FACTORY = new ThreadFactoryBuilder().setNameFormat("worker-%d").build();
+  private static final String DRIVER_CLASS_NAME = "org.postgresql.Driver";
+  private static DSLContext configsDslContext;
+  private static DSLContext jobsDslContext;
+
+  private static final Thread SHUTDOWN_HOOK = new Thread(() -> {
```

Comment:
This SHUTDOWN_HOOK looks like it is currently unused

Reply:
This has been removed/replaced.